models/Sys.java

```diff
@@ -0,0 +1,44 @@
+package tester.models;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+
+public class Sys {
+	private String name, desc;
+	private final LinkedList<Headmate> headmates;
+	private final HashMap<String, Sys> subsystems;
+
+	public Sys(String name) {
+		this.name = name;
+		headmates = new LinkedList<Headmate>();
+		subsystems = new HashMap<String, Sys>();
```

Comment:
unsure why the HashMap value type is Sys, is the intent to create multiple systems, and be able to call them through the subsystem name?

Reply:
Ah, yeah, that was a temp line. Can get rid of. But basically, if you are going to have lookup based on name, it's easier to have a HashMap rather than iterating through to find it

Reply:
true, thanks. we think its a good idea actually